packages/amplify_api/lib/src/graphql/paginated_result_impl.dart

```diff
@@ -28,6 +45,11 @@ class PaginatedResultImpl<T extends Model> extends PaginatedResult<T> {
     return '';
   }
 
+  @override
+  ModelIdentifier get modelIdentifier {
```

Comment:
Why would this not be a list of model identifiers?

Reply:
Or why does it need it at all?

Reply:
Forgot to respond to this question. This dummy getter is added because `PaginatedResultImpl` extends from `Model`, added this following the original impl of `getId` in this class. Been syncing up with @ragingsquirrel3 regarding this part, this may be refactored when finalize custom PK support in API category.